tests/test_statevec_backend.py

```diff
@@ -0,0 +1,52 @@
+import time
+import unittest
+from copy import deepcopy
+
+import numpy as np
+from graphix.ops import States
+from graphix.sim.statevec import Statevec, meas_op
+
+
+class TestStatevec(unittest.TestCase):
```

Comment:
is this supposed to be complete testing of the backend, or is this only for `truncate` method?

Reply:
supposed to be complete testing of the statevector backend. In the future, when we add another method into statevec, we will make another tests on this file.